CONTRIBUTING.md

```diff
@@ -109,6 +128,12 @@ To start server:
 uvicorn saleor.asgi:application --reload
 ```
 
+To start Celery worker:
+
+```shell
+celery --app=saleor.celeryconf:app worker --beat
```

Comment:
This would not work correctly as we're using a custom scheduler to gather the tasks (after first cycle, the celery beat will just stop working).  The proper way of setting up the celery beat is described here: https://docs.saleor.io/developer/running-saleor/task-queue#periodic-tasks

Reply:
I've removed it, will add a separate beat command to poe setup